wgpu-hal/src/dx12/command.rs

```diff
@@ -290,22 +300,21 @@ impl crate::CommandEncoder for super::CommandEncoder {
     }
     unsafe fn discard_encoding(&mut self) {
         if let Some(list) = self.list.take() {
-            if list.close().into_result().is_ok() {
+            if unsafe { list.Close() }.is_ok() {
                 self.free_lists.push(list);
             }
         }
     }
     unsafe fn end_encoding(&mut self) -> Result<super::CommandBuffer, crate::DeviceError> {
         let raw = self.list.take().unwrap();
-        raw.close()
-            .into_device_result("GraphicsCommandList::close")?;
+        unsafe { raw.Close() }.into_device_result("GraphicsCommandList::close")?;
         Ok(super::CommandBuffer { raw })
     }
     unsafe fn reset_all<I: Iterator<Item = super::CommandBuffer>>(&mut self, command_buffers: I) {
         for cmd_buf in command_buffers {
             self.free_lists.push(cmd_buf.raw);
         }
-        self.allocator.reset();
+        let _todo_handle_error = unsafe { self.allocator.Reset() };
```

Comment:
:eyes:

Reply:
I'm planning to resolve this one in the upcoming PR as well.